src/fontra/views/editor/scene-controller.js

```diff
@@ -1462,3 +1524,66 @@ function splitLocation(location, glyphAxes) {
 
   return { fontLocation, glyphLocation };
 }
+
+export function connectTwoDistinctContours(path, firstPointIndex, secondPointIndex) {
```

Comment:
Suggest to call this function `joinContours`.

Reply:
done.